flink-runtime/src/main/java/org/apache/flink/runtime/rpc/registration/RetryingRegistration.java

```diff
@@ -197,7 +197,7 @@ public void onSuccess(Gateway result) {
 				@Override
 				public void onFailure(Throwable failure) {
 					if (!isCanceled()) {
-						log.warn("Could not resolve {} address {}, retrying...", targetName, targetAddress);
+						log.warn("Could not resolve {} address {}, retrying...", targetName, targetAddress, failure);
```

Comment:
I think you can according to http://stackoverflow.com/a/6374166/4815083 and I also just tested it.